stix_shifter_modules/crowdstrike_logscale/configuration/lang_en.json

```diff
@@ -7,17 +7,17 @@
         "help": {
             "label": "Need additional help?",
             "description": "More details on the data source setting can be found in the specified link"
+        },
+        "repository": {
+            "label": "Repository",
+            "description": "The container in which log events collected from various data sources are stored."
```

Comment:
"A container storing log events collected from various data sources." 

Sounds a bit better to me.